src/molecule/provisioner/ansible.py

```diff
@@ -947,7 +947,7 @@ def _verify_inventory(self):
         :return: None
         """
         if not self.inventory:
-            msg = "Instances missing from the 'platform' " "section of molecule.yml."
```

Comment:
@Ruchip16 why do we need a concatenation here?
It can be a single string right?
Am I missing something?